frontend/src/components/LearnPageLayout.tsx

```diff
@@ -21,11 +21,11 @@ export const LearnPageLayout: FC<LearnPageLayoutProps> = ({ children, header, br
       <h1 className="mb-10 rounded-3xl bg-gray-surface px-4 py-6 font-display text-4xl font-medium text-primary-700 md:mb-12 md:px-24 md:py-16 md:text-5xl md:font-bold">
         {header}
       </h1>
-      <div className="grid gap-6 lg:grid-cols-12">
+      <div className="sm:grid sm:gap-6 lg:grid-cols-12">
```

Comment:
Why only grid with gap-6 on `sm:` breakpoint?

Reply:
On mobile I couldn't figure out how to get the table contents from overflowing horizontally.  In Bonnie's case study there's a huge table that will definitely need scrolling.  What I have here works, not sure if it's the best approach though.